packages/@aws-cdk/aws-lambda-event-sources/lib/sqs.ts

```diff
@@ -45,7 +45,7 @@ export interface SqsEventSourceProps {
    *
    * @default - None
    */
-  readonly filters?: Array<{[key: string]: any}>;
+  readonly filters?: Array<lambda.FilterCriteria>;
```

Comment:
I don't think we can do this since it would break existing users. We would need to deprecate this and add a new property.

But I'm not sure I see how this adds much benefit to the user. In the end they still need to provide a `{[key: string]: any}`

Before:
```ts
filters: [{
  Key: 'Value',
}]
```
After:
```ts
filters: [lambda.FilterCriteria.filter({
  Key: 'Value',
})],
```

Reply:
oh I see, you added the `filters` property in the previous PR with the wrong type. Unfortunately I do think that you will need to deprecate `filters` and add a new property since changing the type would be a breaking change.

Reply:
I deprecated `filters` and change to `filterCriteria`